packages/components/src/internal/util/helpLinks.tsx

```diff
@@ -136,3 +137,119 @@ export const JavaDocsLink: FC<JavaDocsLinkProps> = memo(props => {
+                <tr>
+                    <td>yyyy-MM-dd HH:mm</td>
+                    <td>2024-08-14 13:45</td>
+                </tr>
+                <tr>
+                    <td>yyyy-MMM-dd hh:mm a</td>
+                    <td>2024-Aug-14 01:45 PM</td>
+                </tr>
+                <tr>
+                    <td>dd-MMM-yyyy</td>
+                    <td>14-Aug-2024</td>
+                </tr>
+                <tr>
+                    <td>dd-MMM-yy</td>
+                    <td>14-Aug-24</td>
+                </tr>
+            </tbody>
+        </table>
+    </>
+);
+const DATE_HELP_CONTENT = (
+    <>
+        <p>Examples for August 14, 2024:</p>
+        <table className="table-bordered margin-bottom help-table">
+            <thead>
+                <tr>
+                    <th>Format String</th>
+                    <th>Display Result</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>yyyy-MM-dd</td>
+                    <td>2024-08-14</td>
+                </tr>
+                <tr>
+                    <td>yyyy-MMM-dd</td>
+                    <td>2024-Aug-14</td>
+                </tr>
+                <tr>
+                    <td>dd-MMM-yyyy</td>
+                    <td>14-Aug-2024</td>
+                </tr>
+                <tr>
+                    <td>dd-MMM-yy</td>
+                    <td>14-Aug-24</td>
+                </tr>
+            </tbody>
+        </table>
+    </>
+);
+const TIME_HELP_CONTENT = (
+    <>
+        <p>Examples for 01:45:15 PM:</p>
+        <table className="table-bordered margin-bottom help-table">
+            <thead>
+                <tr>
+                    <th>Format String</th>
+                    <th>Display Result</th>
+                </tr>
+            </thead>
+            <tbody>
+                <tr>
+                    <td>HH:mm:ss</td>
+                    <td>13:45:15</td>
+                </tr>
+                <tr>
+                    <td>HH:mm</td>
+                    <td>13:45</td>
+                </tr>
+                <tr>
+                    <td>HH:mm:ss.SSS</td>
+                    <td>13:45:15.000</td>
+                </tr>
+                <tr>
+                    <td>hh:mm a</td>
+                    <td>01:45 PM</td>
+                </tr>
+            </tbody>
+        </table>
+    </>
+);
+
+export const getFolderDateTimeHelpBody = (isDate: boolean = true, isTime: boolean = true): ReactNode => {
+    const content = isDate && isTime ? DATETIME_HELP_CONTENT : isDate ? DATE_HELP_CONTENT : TIME_HELP_CONTENT;
+    const contentLabel = isDate && isTime ? 'date or time' : isDate ? 'date' : 'time';
+    return (
+        <>
+            <p>
+                To control how a {contentLabel} value is displayed, provide a string format compatible with the Java{' '}
+                <JavaDocsLink urlSuffix="java/text/SimpleDateFormat.html">SimpleDateFormat</JavaDocsLink> class.
```

Comment:
as mentioned in chat, we should update this as the user isn't "providing" a string (more like "selecting"). Also, no need to refer to the JavaDocs anymore so can probably drop that part.

Reply:
fixed